docker/config/executor.yml

```diff
@@ -65,3 +65,8 @@ datawave:
       logStatusPeriodMs: 600000
       logStatusWhenChangedMs: 300000
       queryMetricsUrlPrefix: https://localhost:8543/querymetric/v1/id/
+
+hazelcast:
```

Comment:
Can we move this configuration to application-query.yml.  Then you can remove it from both executor.yml and query.yml because both the query and executor service depend on application-query.yml.

Can you also make the cluster name an environment variable (defaulting to the shared cache) so that we can decide which cache to use at docker compose runtime?

For instance, instead of what you have, can you change it to this?
```
hazelcast:
  client:
    clusterName: ${QUERY_CACHE:cache}
  clusterName: ${QUERY_CACHE:cache}
```

That way we can use the original cache with this command:
`docker compose up -d`

And the alternate cache with this command:
`QUERY_CACHE=cache2 docker compose up -d`